Library/Homebrew/brew.sh

```diff
@@ -304,7 +305,8 @@ update-preinstall() {
   # Allow auto-update migration now we have a fix in place (below in this function).
   export HOMEBREW_ENABLE_AUTO_UPDATE_MIGRATION="1"
 
-  if [[ "$HOMEBREW_COMMAND" = "install" || "$HOMEBREW_COMMAND" = "upgrade" || "$HOMEBREW_COMMAND" = "tap" ]]
+  echo HOMEBREW_COMMAND_ARGS=${HOMEBREW_COMMAND_ARGS[0]}
```

Comment:
I assume this `echo` statement was for debugging; it's probably best to remove it for the PR